### ISSUE TITLE
MuxBroadcast widget - PlatformException

### ISSUE THREAD

**Issue by cloudcerebro** [FlutterFlow/flutterflow-issues#283]
Please **thumbs up** this issue if you have also experienced it. You may also add more information if there is something relevant that was not mentioned. However, please refrain from comments that are not constructive, like "I have this problem too", etc.

## Expected behavior
Loads up the MuxBroadcast widget and "Start Streaming" button is displayed.

## Current behavior
On entering the page which contains the MuxBroadcast widget, an Exception occured.

**PlatformException
Failed to create codec for: {mime=video/avc, width=720, height=1280}**

> $_threadException.code: "failed_to_create_live_stream"
> $_threadException.details: null
> $_threadException.message: "Failed to create codec for: {mime=video/avc, width=720, height=1280}"
> $_threadException.stacktrace: null
> $_threadException.hashCode: 324360079
> $_threadException.runtimeType: Type (PlatformException)
> $_threadException.runtimeType.hashCode: 984286987

## To Reproduce
1. Create new page (Eg. StreamPage) with event (Document) parameter
2. Add MuxBroadcast widget
3. Navigate to StreamPage from EventPage passing the parameter, by clicking a button.
4. Export code
5. Run using VS code debugger, on Android Emulator (Pixel 6 Pro API 30)

### Bug Report Code 
IT40iMjfsI52ocMA+KqIbcdR/WQ5KUg8ba8RksFraC4tF4yqD7AMefTBVEhVUsvidgl2ImH8mn8d/t2Pm9z2U8I6GwqVfIB7zZABQwzgbEiQbpiYBs/hY0dwSJYGSSDy

## Context 
MuxBroadcast widget was last seen working on December 23, 12pm (IST).

Exception started occuring when checking on December 26, 11:30am (IST).

## Environment

* Version of FlutterFlow used: 3.0 (Released on December 27, 2022
* Platform (e.g. Web, MacOS Desktop): Android
* Browser name and version: Android Emulator (Pixel 6 Pro API 30)
* Operating system and version (desktop or mobile): mobile

**Comment by bulgariamitko** [comment]
Is this happening at a brand new project as well?

**Comment by cloudcerebro** [comment]
Yes, the same issue happens in a new project as well. I've made the new project public.
Here's the link:
[https://app.flutterflow.io/project/broadcast-rigz1m](url)

**Comment by agreaves** [comment]
Hi @cloudcerebro, thanks for this bug report! I've followed the steps and even downloaded and run the public project you provided and I was not able to reproduce this issue. I initially thought it might be an Android Emulator issue because it may not have all the required functionality (it's worth trying on a physical device where I'd expect it to work). However, I tested on a Pixel 5 API 33 and it worked just fine.

So it could be the Android API (perhaps 30 is not supported), or some other issue. It looks like another user of the package has faced a similar issue: https://github.com/apivideo/api.video-flutter-live-stream/issues/21.

The creator of the package suggests it could be due to the FPS, with 30 being too high for the device. If you choose a lower FPS does this affect the error here?
